schema_based_on_diagram.sql

```diff
@@ -1,23 +1,25 @@
 CREATE TABLE patients(
-	id SERIAL NOT NULL PRIMARY KEY,
-	name VARCHAR(250) NOT NULL,
-	date_of_birth DATE
+    id SERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(250) NOT NULL,
+    date_of_birth DATE
 );
 
 CREATE TABLE medical_histories(
-	id SERIAL NOT NULL PRIMARY KEY,
-	admitted_at TIMESTAMP,
-	patient_id INT 
-	REFERENCES patients(id) ON DELETE CASCADE,
-	status VARCHAR(250) NOT NULL
+id SERIAL NOT NULL PRIMARY KEY,
+admitted_at TIMESTAMP,
+patient_id INT
+REFERENCES patient(id) ON DELETE CASCADE,
+status VARCHAR(250) NOT NULL
 );
 
+
 CREATE TABLE treatments (
 	id SERIAL NOT NULL PRIMARY KEY,
 	type VARCHAR(250) NOT NULL,
 	name VARCHAR(250) NOT NULL
 );
 
+
```

Comment:
Hi @Nejjari001 and @RashidMenhas 
Great job so far.

- [ ] Please all FOREIGN KEYS should have indexes created.

* `CREATE INDEX ON medical_histories (patient_id);`
* `CREATE INDEX ON invoices (medical_history_id);`
* `CREATE INDEX ON invoice_items (invoice_id);`
* `CREATE INDEX ON invoice_items (treatment_id);`
* `CREATE INDEX ON medical_histories_has_treatments (medical_history_id);`
* `CREATE INDEX ON medical_histories_has_treatments (treatment_id);`